animation_nodes/nodes/mesh/mesh_object_input.py

```diff
@@ -6,6 +6,7 @@
 class MeshObjectInputNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_MeshObjectInputNode"
     bl_label = "Mesh Object Input"
+    bl_width_default = 145
```

Comment:
I would prefer if we keep the default width here as well.

Reply:
Now it has the default width.